lib/fittings/swagger_render.js

```diff
@@ -1,11 +1,26 @@
+// Copyright 2016, EMC, Inc.
+
 'use strict';
 
+var injector = require('../../index.js').injector;
+
 module.exports = function create(fittingDef, bagpipes) {
     return function swagger_render(context, next) {
         if (!context.response.headersSent) {
             var status = context.request.swagger.options.success || 200;
-            context.response.status(status).json(context.response.body);
+            var swag = injector.get('Http.Services.Swagger');
```

Comment:
@dalebremner I'd move this injector call outside the swagger_render function.  That way it only needs to be injected when the fitting is created, rather than each time the fitting is called.